### PR TITLE
Fix check for Durbin test

### PR DESCRIPTION
**Pull request by JohnnyDoorn**
Detecting when to use Durbin test was too permissive, as reported here: https://forum.cogsci.nl/discussion/comment/24225#Comment_24225

### CODE REVIEW

File: R/anovarepeatedmeasures.R [hunk 1]
```diff
@@ -1412,8 +1412,9 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
     blocks <- as.factor(longData[, betweenTerms.base64])
     y <- longData[, .BANOVAdependentName]
 
-    useDurbin <- any(table(groups, blocks) != 1)
-
+    # useDurbin <- any(table(groups, blocks) != 1)
+    useDurbin <- any(table(groups, blocks) < 1) & length(unique(table(groups))) == 1
```
Comment (vandenman) [inline R/anovarepeatedmeasures.R:1416]:
```suggestion
    useDurbin <- any(table(groups, blocks) < 1) && length(unique(table(groups))) == 1
```

**Comment by vandenman** [comment]
image

I think you also need to update one unit test

**Comment by boutinb** [comment]
Should this fix be in 0.15?

**Comment by JohnnyDoorn** [comment]
Ideally yes, it's very small and reviewed by don so should be fine I hope!

**Review by vandenman — APPROVED** [review]
Looks good to me!

**Pull request merged**